cluster-autoscaler/processors/nodegroupset/compare_nodegroups.go

```diff
@@ -31,9 +31,11 @@ const (
 	// MaxFreeDifferenceRatio describes how free resources (allocatable - daemon and system pods)
 	// can differ between groups in the same NodeGroupSet
 	MaxFreeDifferenceRatio = 0.05
-	// MaxMemoryDifferenceInKiloBytes describes how much memory
-	// capacity can differ but still be considered equal.
-	MaxMemoryDifferenceInKiloBytes = 256000
+)
+
+var (
+	// MaxMemoryDifference describes how much memory capacity can differ but still be considered equal.
+	MaxMemoryDifference = resource.MustParse("256Mi")
```

Comment:
Moving discussion from https://github.com/openshift/kubernetes-autoscaler/pull/152#discussion_r422887393 here.

Wouldn't this MaxMemoryDifference also apply to much more smaller instances to the point of making the check loosing its value?
i.e If the possible diff range increase with the instance size, should we may be make our tolerance window a percentage of the given total size?

Reply:
Yes it would. For this example the difference was just over 1%, so perhaps making it percentage based would be better. I think we will need to investigate a few more real world examples to come up with a sensible way to do this